tests/TestRender.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace phpdotnet\phd;
 
-class TestRender {
+class TestRender extends Render {
```

Comment:
Either this, or pass a Rendering object in the constructor I guess.

Reply:
Composition over inheritance? :-) I'll make a note to fix this and will keep this in mind for future PRs.